packages/contracts-bedrock/test/vendor/eas/EAS.t.sol

```diff
@@ -0,0 +1,3080 @@
+    Signature
+} from "src/vendor/eas/IEAS.sol";
+import { ISchemaResolver } from "src/vendor/eas/resolver/ISchemaResolver.sol";
+import { Predeploys } from "src/libraries/Predeploys.sol";
+import { EIP712 } from "@openzeppelin/contracts/utils/cryptography/draft-EIP712.sol";
+import { ISemver } from "interfaces/universal/ISemver.sol";
+
+/// @dev Helper contract for testing EIP712 signature verification
+contract TestEIP712Helper is EIP712 {
+    constructor() EIP712("EAS", "1.3.0") { }
+
+    /// @dev Exposes internal hash function for testing
+    function hashTypedDataV4(bytes32 structHash) public view returns (bytes32) {
+        return _hashTypedDataV4(structHash);
+    }
+}
+
+/// @dev Proxy contract for testing EIP712 signature verification
+contract TestEIP712Proxy is EIP712 {
+    address public verifyingContract;
+    string private _name;
+
+    constructor(address _verifyingContract, string memory name_) EIP712(name_, "1.3.0") {
+        verifyingContract = _verifyingContract;
+        _name = name_;
+    }
+
+    /// @dev Returns the name of the contract
+    function name() public view returns (string memory) {
+        return _name;
+    }
+
+    /// @dev Returns the domain separator for EIP712
+    function DOMAIN_SEPARATOR() public view returns (bytes32) {
+        return _domainSeparatorV4();
+    }
+
+    /// @dev Exposes internal hash function for testing
+    function hashTypedDataV4(bytes32 structHash) public view returns (bytes32) {
+        return _hashTypedDataV4(structHash);
+    }
+
+    /// @dev Forwards attestation request to the verifying contract
+    function attestByDelegation(DelegatedAttestationRequest calldata request) external returns (bytes32) {
+        return IEAS(verifyingContract).attestByDelegation(request);
+    }
+}
+
+// Mainnet Contract
+contract EASTest is CommonTest {
+    // Constants
+    uint64 constant NO_EXPIRATION = 0;
+    bytes32 constant ZERO_BYTES32 = bytes32(0);
+    bytes32 private constant ATTEST_TYPEHASH = 0xfeb2925a02bae3dae48d424a0437a2b6ac939aa9230ddc55a1a76f065d988076;
+    bytes32 constant REVOKE_TYPEHASH = 0xb5d556f07587ec0f08cf386545cc4362c702a001650c2058002615ee5c9d1e75;
+    bytes32 constant ATTEST_PROXY_TYPEHASH = 0xea02ffba7dcb45f6fc649714d23f315eef12e3b27f9a7735d8d8bf41eb2b1af1;
+
+    enum SignatureType {
+        Direct,
+        Delegated,
+        DelegatedProxy
+    }
+
+    /// @dev A struct representing the full arguments of the delegated multi attestation request.
+    struct SignatureComponents {
+        uint8 v;
+        bytes32 r;
+        bytes32 s;
+    }
+
+    // Error Selectors
+    error InvalidSchema();
+    error InvalidExpirationTime();
+    error NotFound();
+    error AccessDenied();
+    error InvalidLength();
+    error AlreadyRevokedOffchain();
+    error AlreadyTimestamped();
+    error Irrevocable();
+    error InvalidSignature();
+    error DeadlineExpired();
+
+    // Test State
+    ISchemaRegistry public registry;
```

Comment:
`registry` looks unused. Can you make sure there aren't any unused variables here?